pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -99,8 +99,12 @@ const (
 	// PodAutoscalerConditionReady is set when the revision is starting to materialize
 	// runtime resources, and becomes true when those resources are ready.
 	PodAutoscalerConditionReady = apis.ConditionReady
-	// PodAutoscalerConditionActive is set when the PodAutoscaler's ScaleTargetRef is receiving traffic.
+	// PodAutoscalerConditionActive becomes true when the PodAutoscaler's ScaleTargetRef can receive traffic.
 	PodAutoscalerConditionActive apis.ConditionType = "Active"
+	// PodAutoscalerConditionContainersHealthy becomes true when the containers are up and running.
```

Comment:
Yes, I wanted to make it clear that the condition is for all containers.  `ContainerHealthy` is ambiguous in whether it becomes `true` when any container becomes healthy, or becomes `true` when all containers become healthy.  I didn't rename `ContainerHealthy` in Revision though, because I'm afraid it may break backward compatibility.

Reply:
Revised to Bootstrap condition, so no conflicting in naming anymore.